libiocage/lib/Jail.py

```diff
@@ -499,10 +646,21 @@ def _resolve_name(self, text):
 
         raise libiocage.lib.errors.JailNotFound(text, logger=self.logger)
 
-    def _get_name(self):
+    @property
+    def name(self):
+        """
+        The name (formerly UUID) of the Jail
+        """
         return self.config["id"]
```

Comment:
This is write-only here. Just an abstraction of a jails given identifier. I'm sure we can improve validation in JailConfig.py